components/products/Product.vue

```diff
@@ -0,0 +1,75 @@
+<script setup lang="ts">
+import { Product } from '@prisma/client';
+
+defineProps<{
+  product: Product;
+}>()
+
+</script>
+
+<template>
+  <div class="product">
+
+    <img v-if="product.image" :src="product.image" alt="Product Image" class="image" />
```

Comment:
The missing img property might be a better "more specific "alt"?  Otherwise I'm not sure what other important property might be missing.

Malforming of the image could be fixed by setting a width in px. But then when using grid, the elements covered each other in some screen-widths. I fixed it then by using flex in products.

Reply:
Hmm, setting a CSS `width` might not be the best idea when, for example, the image is of a lesser quality (let's say 200x100) and we would try to set it for the entire container which might be bigger - the quality of the image would be destroyed.

I was trying to lead you to setting an `object-fit: contain` (or smth similar) and more importantly setting `width` and `height` props for the img. You can probably ask chatgpt why that is quite important :)